hack/package.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+USE_DOCKER=${USE_DOCKER:-"0"}
+VERSION=${VERSION:-"0.0.$(date +%s)"}
+
+curDir=$(cd "$(dirname "$0")" && pwd)
+cd "${curDir}" || return
+BUILD_SOURCE_HOME=$(cd ".." && pwd)
+
+. ./env.sh
+
+BUILD_PATH=bin/${GOOS}_${GOARCH}
+DFDAEMON_BINARY_NAME=dfdaemon
+DFGET_BINARY_NAME=dfget
+
+main() {
+    cd "${BUILD_SOURCE_HOME}" || return
+    # Maybe we should use a variable to set the directory for release,
+    # however using a variable after `rm -rf` seems risky.
+    mkdir -p release
+    rm -rf release/*
+
+    if [ "1" == "${USE_DOCKER}" ]
+    then
+        echo "Begin to package with docker."
+        FPM="docker run --rm -it -v $(pwd):$(pwd) -w $(pwd) inoc603/fpm:alpine"
+    else
+        echo "Begin to package in local environment."
+        FPM="fpm"
+    fi
+
+    case "$1" in
+        rpm )
+            build_rpm
+            ;;
+        deb )
+            build_deb
+            ;;
+        * )
+            build_rpm
+            build_deb
+            ;;
+    esac
+}
+
+# TODO: Add description
+DFGET_DESCRIPTION="dfget"
+DFDAEMON_DESCRIPTION="dfdaemon"
+# TODO: Add maintainer
+MAINTAINER="dragonflyoss"
+
+build_rpm() {
+    ${FPM} -s dir -t rpm -f -p release --rpm-os=linux \
```

Comment:
Could we build `dfget` and `dfdaemon` as one package, and create soft links in `/usr/local/bin` after installing them?

Reply:
Yes. I think building them into one package is better, since there are only 2 binaries that are usually used together, and their size are not very big. I build them into 2 packages because it's not clear in #183 how many packages there should be.

How about a `df-client` package for `dfget` and `dfdaemon`，and a `df-supernode` for `supernode` only?

Reply:
OK.

Reply:
Implemented in fa5d61236fd7c5ca943acade97435c32f3410564